src/main/cpp/ToxCore/filetransfer.cpp

```diff
@@ -133,3 +133,19 @@ TOX_METHOD (void, FileSendChunk,
     tox_file_send_chunk, friendNumber, fileNumber, position, chunkData.data (), chunkData.size ()
   );
 }
+
+
+/*
+ * Class:     im_tox_tox4jToxCoreImpl
+ * Method:    toxHash
+ * Signature: ([B)[B
+ */
+TOX_METHOD (jbyteArray, Hash,
```

Comment:
Are file transfers the only use cases for tox_hash (and will it stay that way in the future)? If not, this should be moved to a different (new) file.